packages/pwa/app/components/product-tile/index.jsx

```diff
@@ -70,7 +82,13 @@ const ProductTile = (props) => {
         >
             <Box {...styles.imageWrapper}>
                 <AspectRatio {...styles.image}>
-                    <Img alt={image.alt} src={image.disBaseLink} />
+                    <Img
+                        {...getImageProps({
+                            alt: image.alt,
+                            src: image.disBaseLink || image.link,
```

Comment:
Will this logic result in the standard link having all the dis transformations on it?

Reply:
@bendvc with my latest changes, the `src` value is no longer transformed, regards if it's a `disBaseLink` or `link`.